dev-build/conf/routes

```diff
@@ -105,6 +105,7 @@ GET            /top-stories.json
 GET            /top-stories/trails.json                                                                                          controllers.TopStoriesController.renderTrails()
 GET            /related/*path.json                                                                                               controllers.RelatedController.render(path)
 GET            /related/*path                                                                                                    controllers.RelatedController.renderHtml(path)
+GET            /people-who-read/:testVariant/*path.json                                                                          controllers.RelatedController.renderPeopleWhoRead(path, testVariant)
```

Comment:
Have you added this to the onwards routes as well?
(Or are you just running it locally)